Taxopark/Taxopark/appsettings.json

```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "mysqlconnection": {
+    "connectionString": "server=db;port=3306;userid=dbuser;password=dbuserpassword;database=taxopark;"
```

Comment:
It will be more secure when password will not be stored in any settings/config files.

Reply:
A user password was removed.